plugin/src/main/java/com/playmonumenta/scriptedquests/timers/CommandTimer.java

```diff
@@ -47,14 +49,31 @@ public CommandTimer(Plugin plugin, int period) {
 		} else {
 			mPeriodStr = ChatColor.BLUE + Integer.toString(mPeriod) + "t";
 		}
-
-
 	}
 
 	/********************************************************************************
 	 * Public Methods
 	 *******************************************************************************/
 
+	public void unloadAllInWorld(World world) {
+		Iterator<Map.Entry<UUID, CommandTimerInstance>> iter = mTimers.entrySet().iterator();
+		while (iter.hasNext()) {
+			Map.Entry<UUID, CommandTimerInstance> entry = iter.next();
+			UUID uuid = entry.getKey();
+			CommandTimerInstance timer = entry.getValue();
+
+			if (entry.getValue().getWorld().equals(world)) {
+				mCoords.remove(timer.getCoords());
+				timer.unload(mPlugin);
+				iter.remove();
+
+				for (LinkedHashMap<UUID, CommandTimerInstance> map: mTickTimers) {
```

Comment:
I bet there's a better way to do this, like checking which map to use from the timer we're removing. Good enough for a hotfix, though.

Reply:
Agreed. Needs doing for unload() too, which this is copied from.